src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -528,15 +662,30 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
         );
     }
 
-    function test_revert_operatorAllocated_notActive() public {
+    /**
+     * Attempts to slash an operator before the allocation is active
+     * Validates:
+     * 1. The events of the slash indicate nothing was slashed
+     * 2. Storage is not mutated post slash
+     * 3. The operator's allocation takes effect as normal post slash
+     */
+    function test_operatorAllocated_notActive() public {
         AllocateParams[] memory allocateParams = _newAllocateParams(defaultOperatorSet, WAD);
 
         cheats.prank(defaultOperator);
-        allocationManager.modifyAllocations(defaultOperator, allocateParams);
+        allocationManager.modifyAllocations(defaultOperator, allocateParams);    
+
+        uint64 encumberedMagnitudeBefore = allocationManager.encumberedMagnitude(defaultOperator, strategyMock);
+        uint64 maxMagnitudeBefore = allocationManager.getMaxMagnitudes(defaultOperator, strategyMock.toArray())[0];
+
+        // The only slash event we expect is the OperatorSlashed. There isn't a way to validate
```

Comment:
Was thinking about this too when writing DM unit tests as well and wanting to check for certain events
not being emitted (when not delegated for example).
Perhaps in a separate PR we can write a utils library using 
`vm.recordLogs()` and `vm.getRecordedLogs().length` before and after the call

Reply:
100% , let me add to TODOs

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/911/commits/03aa2f85a712f56093d9238934b5accb79e83a53